client/android/app/src/main/java/com/snapscreen/MainApplication.java

```diff
@@ -24,6 +25,7 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
               new MainReactPackage(),
+            new ReactNativeConfigPackage(),
```

Comment:
I would expect you to have the strongest opinion about our Java so if you're ok with the Google style guide then I am too 👍